bundles/automation/org.eclipse.smarthome.automation.providers/src/main/java/org/eclipse/smarthome/automation/internal/core/provider/AutomationResourceBundlesTracker.java

```diff
@@ -84,8 +85,15 @@ protected void deactivate(BundleContext bc) {
         rImporter = null;
     }
 
+    @SuppressWarnings({ "rawtypes" })
+    protected void setProvider(Provider provider) {
```

Comment:
As you have choosen 0..n as cardinality wouldn't be "add" be more precise then "set"?